src/test/java/org/springframework/data/elasticsearch/core/paginating/SearchAfterIntegrationTests.java

```diff
@@ -15,13 +15,16 @@
  */
 package org.springframework.data.elasticsearch.core.paginating;
 
-import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import org.elasticsearch.search.sort.SortBuilders;
+import org.elasticsearch.search.sort.SortOrder;
```

Comment:
This pulls in dependencies to Elasticsearch classes in this test, this test is independent of the used query type.